app/models/seeker.rb

```diff
@@ -20,7 +20,8 @@ def self.search(query, baseproject, project = nil, exclude_filter = nil, exclude
       words = query.split(" ").reject { |part| part.match(/^[0-9_\.-]+$/) }
       versrel = query.split(" ").select { |part| part.match(/^[0-9_\.-]+$/) }
       logger.debug "splitted words and versrel: #{words.inspect} #{versrel.inspect}"
-      raise InvalidSearchTerm.new "Please provide a valid search term" if words.blank? && project.blank?
+      raise InvalidSearchTerm.new "Please provide a valid search term" if words.blank? && versrel.blank?
+      raise InvalidSearchTerm.new "The package name is required when searching for a version" if words.blank? && !versrel.blank?
```

Comment:
Nitpick: I think `versrel.present?` reads a little better than `!versrel.blank?`:
```suggestion
      raise InvalidSearchTerm.new "The package name is required when searching for a version" if words.blank? && versrel.present?
```